dash-pipeline/SAI/specs/dash_tunnel.yaml

```diff
@@ -49,6 +49,32 @@ sai_apis:
     valid_only: null
     is_vlan: false
     deprecated: false
+  - !!python/object:utils.sai_spec.sai_attribute.SaiAttribute
+    name: SAI_DASH_TUNNEL_ATTR_SIP
+    description: Action parameter sip
+    type: sai_ip_address_t
+    attr_value_field: ipaddr
+    default: 0.0.0.0
+    isresourcetype: false
+    flags: CREATE_AND_SET
+    object_name: null
+    allow_null: false
+    valid_only: null
+    is_vlan: false
+    deprecated: false
+  - !!python/object:utils.sai_spec.sai_attribute.SaiAttribute
+    name: SAI_DASH_TUNNEL_ATTR_MAX_MEMBER_SIZE
+    description: Action parameter max member size
+    type: sai_uint32_t
+    attr_value_field: u32
+    default: '1'
+    isresourcetype: false
+    flags: CREATE_AND_SET
```

Comment:
This should be CREATE_ONLY. Max members should not be allowed to be changed after group create.

Reply:
yep and done, along with the tunnel member attribute flags.

Reply:
Can we please make the SAI_DASH_TUNNEL_ATTR_DASH_ENCAP and SAI_DASH_TUNNEL_ATTR_TUNNEL_KEY create-only as well ?